packages/message-pickup-repository-dynamodb/tests/dynamoDbMessagePickupRepository.test.ts

```diff
@@ -0,0 +1,47 @@
+import { randomUUID } from 'node:crypto'
+import { expect, suite, test } from 'vitest'
+import { DynamoDbMessagePickupRepository } from '../src'
+
+suite('dynamoDbMessagePickupRepository', () => {
+  let repository: DynamoDbMessagePickupRepository
+  const connectionId = randomUUID()
+  let messageId: string
+
+  test('instantiate', async () => {
+    repository = await DynamoDbMessagePickupRepository.initialize({
+      dynamoDbRepositoryOptions: { region: 'local', credentials: { accessKeyId: 'local', secretAccessKey: 'local' } },
```

Comment:
Since this is already `DynamoDbMessagePickupRepository`, i think the param could you be `options` or something like `database`, or `connection`.